README.md

```diff
@@ -11,20 +11,29 @@ Dependencies:
 ```sh
 pipenv install
 ```
+```sh
+npm install -g sass
+```
 
 ### Initialize Database
 ```sh
 pipenv run python manage.py migrate
 ```
 
+### Compile your CSS

```

Comment:
Is having scss _really_ necessary? It appears that you're mostly using good ol' CSS features +  just compiling bootstrap, which you could just vendor a pre-compiled, minified and optimised version of. 

Adding extra things to the pipeline are going to slow down other engineers working on this project, and make it more difficult to maintain over time. I would avoid if possible.